backend/src/core/services/whatsapp.service.ts

```diff
@@ -0,0 +1,8 @@
+import WhatsAppClient from '@shared/clients/whatsapp-client.class'
+import config from '@core/config'
+
+const whatsappClient = new WhatsAppClient(config.get('whatsapp'))
```

Comment:
A neater implementation (imo) is to do as below here
```
export const whatsappClient = ...
```
then in `services/index.ts` do
```ts
export * as WhatsAppService from './whatsapp.service.ts'
```

Reply:
ohh I was following how it's done for MailService. I won't make this change in the interest of consistency?
![image](https://github.com/opengovsg/postmangovsg/assets/67887489/857b49fa-8d37-4bff-a3c2-07f1b4343d2f)
![image](https://github.com/opengovsg/postmangovsg/assets/67887489/bfa1f06c-7253-4902-99f2-c27089c4eeec)

Reply:
actually the re-exporting in the index.ts is redundant if I'm not wrong